src/partials/append-task.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable import/prefer-default-export */
+import { DataBase } from './storage';
+import { updateStatus } from './update-status';
+
+const taskBox = (newTask, taskArea) => {
+  const task = document.createElement('div');
+  task.classList.add('task');
+  const taskDescription = document.createElement('div');
+  taskDescription.classList.add('task-description');
+  const titleTask = document.createElement('h3');
+  titleTask.innerHTML = `<b>${newTask.taskName}</b>`;
+  const descriptionTask = document.createElement('p');
+  descriptionTask.innerHTML = `${newTask.description}`;
+  const dateTask = document.createElement('p');
+  dateTask.innerHTML = `Due date: ${newTask.date}`;
+  const statusTask = document.createElement('p');
+  taskDescription.appendChild(titleTask);
+  taskDescription.appendChild(descriptionTask);
```

Comment:
From this file's name, we expect to see a pure method to append a new task. Yes there's but there also DOM manipulation in same file. This is due to organisation issues i was saying above:

From ODIN project guidelines, logic and DOM manipulation should be separate
![Screen Shot 2019-11-20 at 7 10 56 PM](https://user-images.githubusercontent.com/40712459/69264726-2b893b00-0bc8-11ea-8bbd-dca7c32be662.png)

We can see this in other files